api/inc/debug_exports.h

```diff
@@ -19,12 +19,19 @@
 
 #include "api/inc/halt_exports.h"
 #include <stdint.h>
+#include "api/inc/vmpu_exports.h"
 
-/* Debug box driver -- Version 0
+
+#define UVISOR_DEBUG_BOX_VERSION    (1)
+
+
+/* Debug box driver -- Version 1
```

Comment:
Remove ` -- Version 1`. Let the `UVISOR_DEBUG_BOX_VERSION` be the only source of version, so it's easy to update.